<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlamegraphSettingsPage</class>
 <widget class="QWidget" name="FlamegraphSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="userPathsLabel">
     <property name="text">
      <string>User Paths:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KEditListWidget" name="userPaths" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="systemPathsLabel">
     <property name="text">
      <string>System Paths:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KEditListWidget" name="systemPaths" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
